docs/examples.md

```diff
@@ -9,35 +9,25 @@ duplicating the UI. When that's not possible, a workaround is to match on
 the first element.
 
 ```swift
-// Swift Custom Matcher
-/**
- *  Example Usage:
- *
- *  EarlGrey.select(elementWithMatcher:grey_allOfMatchers([
- *    grey_accessibilityID("some_id"),
- *    grey_interactable(),
- *    grey_firstElement()])).assert(grey_notNil())
- *
- *  Only intended to be used with selectElementWithMatcher.
- */
+/// Example Usage:
+///
+///     EarlGrey.select(elementWithMatcher: grey_allOf([
+///       grey_accessibilityID("some_id"),
+///       grey_interactable(),
```

Comment:
Can you move grey_interactable() after grey_firstElement() ?

Reply:
Done.